scan/cli.go

```diff
@@ -173,9 +178,9 @@ func AuditCmd(c *cli.Context) error {
 		case coreutils.Go:
 			err = AuditGoCmd(c)
 		case coreutils.Pip:
-			err = AuditPipCmd(c)
+			err = AuditPythonCmd(c, artifactoryUtils.Pip)
 		case coreutils.Pipenv:
-			err = AuditPipenvCmd(c)
+			err = AuditPythonCmd(c, artifactoryUtils.Pipenv)
```

Comment:
Since the above ```Pipenv``` and ```Pip``` const are now used by the ```audit```, we should probably find a better place for them, than ```artifactoryUtils``` (because audit isn't related to Artifactory).